docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -585,8 +585,9 @@ of your Hazelcast data pipeline.
 
 === Upgrade Your JDK
 
-Hazelcast does not support JDK 8, so if you are still on JDK 8, please upgrade it.
-The garbage collectors of JDK 11+ have been improved to achieve much higher throughputs. 
+Hazelcast does not support JDK 11 and older versions. Newer JDK releases often contain performance improvements, consider upgrading to 
```

Comment:
Maybe instead of mentioning a specific version we could mention our policy of supporting last 2 LTS versions, and list what exactly we support in `versioning-compatibility.adoc`

Reply:
I'm not sure it's a good idea to put our policy buried on one of these pages. If Java 23 LTS comes out tomorrow, we won't be supporting it until the next major release of Hazelcast - whereas if it says we support the latest 2 LTS releases, it would be implied that 23 was supported today.